crowbar_framework/app/models/ceph_service.rb

```diff
@@ -181,6 +192,21 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
       end
     end
 
+    calamari.each do |n|
+      node = NodeObject.find_node_by_name(n)
+      node.crowbar["crowbar"] ||= {}
+      node.crowbar["crowbar"]["links"] ||= {}
+
+      for t in ['public', 'admin'] do
```

Comment:
Good question. Calamari's only used for monitoring the ceph cluster, so probably only wants an address on the admin network, right?